source/inotify.cpp

```diff
@@ -76,6 +76,32 @@ void Inotify::watchFile(const FileSystemEvent& fse)
     mDirectorieMap.emplace(wd, fse.getPath());
 }
 
+void Inotify::watchDirectory(const FileSystemEvent& fse)
+{
+    if (!checkWatchDirectory(fse))
+        return;
+
+    mError = 0;
+    int wd = 0;
+    wd = inotify_add_watch(mInotifyFd, fse.getPath().c_str(), getEventMask(fse.getEvent()));
```

Comment:
```suggestion
    const int wd = wd = inotify_add_watch(mInotifyFd, fse.getPath().c_str(), getEventMask(fse.getEvent()));
```

Reply:
I replaced it in 6886f46, I copied that bit from `watchFile` though so I also changed it there.